src/modules/healtcheck/healtcheck.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Version } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { HealtcheckService } from './healtcheck.service';
+
+@ApiBearerAuth()
+@ApiTags('healtcheck')
+//@Controller('healtcheck')
+@Controller('')
+export class HealtcheckController {
+  constructor(private readonly healtcheckService: HealtcheckService) {}
+
+  @Get('/live')
+  @Version('1')
+  @ApiOperation({ summary: 'live' })
+  @ApiResponse({ status: 200, description: 'Check if this api is running'})
+  live(): any {
+    return {
+      status: 200,
+      message: "Online",
+      datetime: new Date().toISOString()
+    };
+  }
+
+  @Get('healtcheck')
+  @Version('1')
+  @ApiOperation({ summary: 'Alias for /ready' })
+  @ApiResponse({ status: 200, description: 'Check if this api is running'})
+  healtcheck(): any {
+    return this.ready();
+  }
+
+  @Get('ready')
+  @Version('1')
+  @ApiOperation({ summary: 'ready' })
+  @ApiResponse({ status: 200, description: 'Check if this api can access dependencies'})
+  async ready(): Promise<any> {
```

Comment:
Can we avoid using `any` type?

Reply:
I changed to more specific types, but it is far from perfect. Most replacements are to type object. 
I didn't create new classes or interfaces, hope this is good for now.